src/redux/InstructionsSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const InstructionsSlice = createSlice({
+  name: "instructions",
+  initialState: {},
+  reducers: {
+    setInstructions: (_state, action) => {
+      return action.payload;
+    },
+    setCurrentStepPosition: (state, action) => {
+      state.currentStepPosition = action.payload;
+    },
+  },
```

Comment:
🔧 just being picky but it'd be good if this was setup initially so it was fully expandable and testable. I'd shift these reducers into the `./reducers` directory and pull together some quick initial tests. I imagine these are going to get more complicated